src/applications/static-pages/createCoronavirusChatbot.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+export default (store, widgetType) => {
+  // Derive the element to render our widget.
+  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
+
+  ReactDOM.render(
+    <Provider store={store}>
+      <iframe
+        id={'va-coronavirus-chatbot'}
+        title="VA Coronavirus chatbot"
+        src="https://va-covid19-chatbot-dev.azurewebsites.net/chatbot.html"
+        style={{ height: `${800}px`, width: `${100}%` }}
+      />
+    </Provider>,
+    root,
+  );
```

Comment:
If you don't want to worry about writing React, you don't have to. It's just strongly preferred but I understand that this is time-sensitive. I would delete this whole `ReactDOM` along with the React imports.